internal/rest/avi_obj_vs.go

```diff
@@ -457,6 +560,10 @@ func (rest *RestOperations) AviVsCacheAdd(rest_op *utils.RestOp, key string) err
 								if len(vsvip_cache_obj.Vips) > 0 {
 									vip := vsvip_cache_obj.Vips[0]
 									vs_cache_obj.Vip = vip
+									if len(vsvip_cache_obj.Fips) > 0 {
```

Comment:
In case fip for a vs is remove, I think we should set vs_cache_obj.Fip to "". But in this case we would ignore the condition and fip won't be removed.

Reply:
setting the Fip to "" in else condition now